src/meta/src/hummock/manager/mod.rs

```diff
@@ -1759,6 +1761,7 @@ where
             Excluded(old_checkpoint_id),
             Included(new_checkpoint_id),
         ));
+        trigger_stale_ssts_stat(&self.metrics, versioning.ssts_to_delete.len());
```

Comment:
In this pr, we need to `trigger_stale_ssts_stat` after changing the `versioning.ssts_to_delete`,This call is easily missed after we add new logic.
Can we make versioning maintain metrics, and wrap ssts_to_delete with metrics reporting into a function to alleviate the above problem?